src/main/drivers/sonar_srf10.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "platform.h"
+#include "drivers/sonar.h"
+
+// Technical specification is at: http://robot-electronics.co.uk/htm/srf10tech.htm
+#define SRF10_MAX_RANGE_CM 600 // 6m, from SFR10 spec sheet
```

Comment:
Sonar parameters are returned in init - is it necessary to publish them both ways?